index.html

```diff
@@ -13,7 +17,128 @@
   <![endif]-->
 </head>
 
-<body>
-  <h1>My Resume</h1>
-</body>
+  <body>
+    <header>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <a href="https://leagueoflegends.fandom.com/wiki/Jinx">Portfolio</a>
+        <a href="#contact">Contact</a>
+      </nav>
+    </header>
+
+    <div class="topContainer">
+      <img class="selfie" src="img/jinx.png" alt="">
```

Comment:
If you change this to:
```
<div class="selfie">
    <img src="img/jinx.png" alt="">
</div>
```

Check the comment in `index.less` for how to style. This should fix Jynx's profile pic from squishing as the webpage gets smaller '' '